### PR TITLE
Firewall rule: create rule at the top of the list

### PR DESCRIPTION
**Pull request by stephdl**
Screenshot (9)

when you create or duplicate a rule, then you can decide to create it at the top or at the bottom of the list.

If it worth it I can do it for the local-rules

what do you think @gsanchietti 

trello card : https://trello.com/c/6Q5PyQng/219-migliorie-interfaccia-firewall

https://github.com/NethServer/dev/issues/6284

### CODE REVIEW

**Comment by gsanchietti** [comment]
I like the approach, and I think it should fits also inside the Local Rules page.
Let's see what @cotosso thinks about it

**Comment by stephdl** [comment]
Ok local rule done, I changed slightly how to retrieve the array order, I use now javascript

so when you create/duplicate a rule you can decide if you want it at the top or at the bottom of the list

**Comment by cotosso** [comment]
Just tested in "rules" and in "local rules" sections. it can be very useful in complex installations.
Well done @stephdl

**Review by cotosso — APPROVED** [review]
I approve this change

**Comment by gsanchietti** [comment]
So the overall solution is good, please @stephdl open the issue and move the PR to 'ready for review' state

File: ui/public/i18n/language.json [hunk 1]
```diff
@@ -429,7 +429,10 @@
         "udpPorts": "UDP ports",
         "access": "Access",
         "service_edited_successfully": "Service edited successfully",
-        "service_edited_error": "Service not edited"
+        "service_edited_error": "Service not edited",
+        "Order": "Rule position",
+        "top": "create at the top",
```
Comment (edospadoni) [inline ui/public/i18n/language.json:434]: do we use the capital letter? `Create at the top`

File: ui/public/i18n/language.json [hunk 2]
```diff
@@ -429,7 +429,10 @@
...
+        "bottom": "create at the bottom"
```
Comment (edospadoni) [inline ui/public/i18n/language.json:435]: do we use the capital letter? `Create at the bottom`

**Review by edospadoni — CHANGES_REQUESTED** [review]

File: ui/public/i18n/language.json (hunk 1 above)
Comment (gsanchietti) [reply ui/public/i18n/language.json:434]: I suggest to reuse the same labels from NethGUI: https://github.com/NethServer/nethserver-firewall-base/blob/master/root/usr/share/nethesis/NethServer/Language/en/NethServer_Module_FirewallRules.php#L43-L44

Also, I'd change the label key name from `Order` to `order`: all other keys are lowercase :)

Comment (stephdl) [reply ui/public/i18n/language.json]: done

**Comment by stephdl** [comment]
it should be good @edospadoni @gsanchietti 

- label of Nethgui reused
- capital letter
- key renamed with lower case

File: ui/public/i18n/language.json [hunk 3]
```diff
@@ -429,7 +429,10 @@
         "udpPorts": "UDP ports",
         "access": "Access",
         "service_edited_successfully": "Service edited successfully",
-        "service_edited_error": "Service not edited"
+        "service_edited_error": "Service not edited",
+        "order": "Rule position",
+        "Create_first_label": "Create at the top",
```
Comment (edospadoni) [inline ui/public/i18n/language.json:434]: `"Create_first_label": "Create at the top"` the json attribute should be in lower case:
`"create_first_label": "Create at the top"`

File: ui/public/i18n/language.json [hunk 4]
```diff
@@ -429,7 +429,10 @@
...
+        "Create_last_label": "Create at the bottom"
```
Comment (edospadoni) [inline ui/public/i18n/language.json:435]: `"Create_last_label": "Create at the bottom"` the json attribute should be in lower case:
`"create_last_label": "Create at the bottom"`

**Review by edospadoni — CHANGES_REQUESTED** [review]

File: ui/public/i18n/language.json (hunk 4 above)
Comment (stephdl) [reply ui/public/i18n/language.json:435]: it is like in nethgui :D

Comment (stephdl) [reply ui/public/i18n/language.json]: done

File: ui/public/i18n/language.json (hunk 3 above)
Comment (stephdl) [reply ui/public/i18n/language.json]: done

**Review by edospadoni — APPROVED** [review]

**Pull request merged**